java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/utils/UtilityMethods.java

```diff
@@ -0,0 +1,34 @@
+package com.xxAMIDOxx.xxSTACKSxx.utils;
+
+import com.microsoft.azure.spring.data.cosmosdb.core.query.CosmosPageRequest;
+import com.xxAMIDOxx.xxSTACKSxx.api.v1.menu.dto.SearchMenuResultItem;
+import com.xxAMIDOxx.xxSTACKSxx.model.Menu;
+import org.springframework.data.domain.Sort;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+/**
+ * A class containing various utility methods.
+ *
+ * @author ArathyKrishna
+ */
+public class UtilityMethods {
```

Comment:
This will become a dumping ground for all utils. 
Use subject specific classes.